azure-pipelines.yml

```diff
@@ -1,15 +1,12 @@
 variables:
-  # Cannot use key:value syntax in root defined variables
   - name: _TeamName
     value: DotNetCore
   - name: _PublishUsingPipelines
     value: true
   - name: _DotNetArtifactsCategory
     value: .NETCore
-  - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
```

Comment:
you set this pipeline to validate PRs that change this file. should restore `${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:` lines but simplify them to `${{ if notin(variables['Build.Reason'], 'PullRequest') }}:`

some of the comments near the other conditions may be useful to restore as well